src/content/ko/tools/lighthouse/index.md

```diff
@@ -0,0 +1,157 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Learn how to set up Lighthouse to audit and improve your web apps.
+
+{# wf_updated_on: 2017-01-18 #}
+{# wf_published_on: 2016-09-27 #}
+
+# Lighthouse {: .page-title }
+
+<img src="/web/progressive-web-apps/images/pwa-lighthouse.png"
+      class="lighthouse-logo attempt-right" alt="Lighthouse Logo">
+
+<style>
+figure {
+  text-align: center;
+}
+.lighthouse-logo {
+  height: 150px;
+  width: auto;
+}
+.lighthouse-install-container .button-primary {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  padding: 32px;
+  background-color: #2979ff;
+}
+.button-primary.lighthouse-install {
+  background: url('images/lighthouse-icon-128.png') no-repeat 0 50%;
+  background-size: 63px;
+  background-color: #304ffe;
+  padding: 32px 16px 32px 64px;
+}
+.lighthouse-install-container .material-icons {
+  margin-right: 4px;
+  vertical-align: middle;
+}
+.lighthouse-install-container {
+  text-align: center;
+  margin: 32px;
+}
+</style>
+
+[Lighthouse](/web/tools/lighthouse/)는 웹 앱의 품질을 개선하는
+[오픈 소스](https://github.com/GoogleChrome/lighthouse) 자동화 도구입니다.
+
+Lighthouse는 크롬 확장프로그램, 커맨드 라인에서, 노드 모듈에서 프로그램으로도 사용할 수 있습니다.
+Lighthouse에 확인할 URL을 지정하고, 페이지에 대한 테스트를 실행한 다음, 페이지에 대한 보고서를 생성합니다.
+여기에서 실패한 테스트는 앱을 개선하기 위해 할 수 있는 것에 대한 지표로 사용할 수 있습니다.
+
+<p class="lighthouse-install-container">
+  <a class="button button-primary lighthouse-install gc-analytics-event"
+     data-category="lighthouse" data-action="install" data-label="blipmdconlkpinefehnmjammfjpmpbjk"
+     href="https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk"
+     title="Install Lighthouse Chrome Extension" target="_blank">
+    <span class="material-icons">file_download</span>
+    Lighthouse 설치
+  </a>
+  <a class="button button-primary gc-analytics-event"
+     data-category="ligthhouse" data-action="bug"
+     href="https://github.com/GoogleChrome/lighthouse/issues/new"
+     title="File an issue or feature request" target="_blank">
+    <span class="material-icons">bug_report</span>
+    이슈 찾기
+  </a>
+</p>
+
+Note: Lighthouse는 현재 [홈스크린에 추가](/web/fundamentals/engage-and-retain/app-install-banners/)나 오프라인 지원 같은,
+프로그레시브 웹 앱 기능에 중점을 둡니다. 그러나 프로젝트의 가장 중요한 목표는 웹 앱 품질의 모든 측면에 대한 end-to-end 테스트를 제공하는 겁니다.
+
+## Lighthouse 설정하기
+
+Lighthouse는 크롬 확장프로그램, 커맨드 라인 도구로 실행할 수 있습니다.
+크롬 확장프로그램은 리포트를 읽는데 더 유저에 친숙한 인터페이스를 제공합니다.
+커맨드 라인 도구는 Lighthouse를 지속적 통합 시스템에 넣을 수 있습니다.
+
+### 크롬 확장프로그램 설치하기
```

Comment:
The Chrome product name shouldn't be translated, this goes for all instances throughout the document.